### PR TITLE
chore: preparing 1.6.0 release

### PR DESCRIPTION
**Pull request by Mat001**
Updated CHANGELOG and version files.

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 Changes that have landed but are not yet released.	Changes that have landed but are not yet released.
 
+## [1.6.0] - February 16th, 2021
+## New Features
+-  Add new info and error messages to decide reasons ([#304](https://github.com/optimizely/go-sdk/pull/304))
+-  Refactor reasons for Decide API ([#303](https://github.com/optimizely/go-sdk/pull/303))
+-  Add decide, DecideAll and DecideForKeys API ([#299](https://github.com/optimizely/go-sdk/pull/299))
+-  Add support for decide options and decision reasons ([#298](https://github.com/optimizely/go-sdk/pull/298))
+-  Add UserContext ([#297](https://github.com/optimizely/go-sdk/pull/297))
```
Comment (jaeopt) [inline CHANGELOG.MD:16]: Why don't you copy from python CHANGELOG? We can use the same format but different api names.

**Review by jaeopt — CHANGES_REQUESTED** [review]

File: CHANGELOG.MD (hunk 1 above)
Comment (Mat001) [reply CHANGELOG.MD:16]: I could do that, but then that would be inconsistent with the changelog history in go-sdk no?
changelog formats in py and go sdks are different

Comment (Mat001) [reply CHANGELOG.MD]: ok, now updated

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**